internal/horusec-javascript/ast.go

```diff
@@ -285,13 +286,34 @@ func (p *parser) parseStmt(node *cst.Node) ast.Stmt {
 			stmt.Else = p.parseStmt(alt)
 		}
 		return stmt
-	case ElseClause, StatementBlock:
+	case ElseClause, StatementBlock, FinallyClause:
```

Comment:
I think that `FinallyClause` here is not necessary?

Reply:
Done!